tournaments/index.js

```diff
@@ -1097,9 +1100,34 @@ let commands = {
 				return this.errorReply("The tournament's banlist is already empty.");
 			}
 			tournament.banlist = [];
+			tournament.customFormat = tournament.format;
 			this.room.addRaw("<b>The tournament's banlist was cleared.</b>");
+			this.room.send('|tournament|update|' + JSON.stringify({customFormat: tournament.customFormat}));
 			this.privateModCommand("(" + user.name + " cleared the tournament's banlist.)");
 		},
+		name: 'setname',
+		customname: 'setname',
+		setname: function (tournament, user, params, cmd) {
+			if (params.length < 1) {
+				return this.sendReply("Usage: " + cmd + " <comma-separated arguments>");
+			}
+			if (tournament.banlist.length < 1) {
+				return this.errorReply("The tournament cannot be named unless it has a banlist.");
+			}
+			let name = Chat.escapeHTML(params[0].trim());
```

Comment:
Because you don't seem to be changing ``name``, ``const`` would probably also work here.  Should we do that instead maybe?